docs/user-documentation/metadata.md

```diff
@@ -1,15 +1,17 @@
 # Metadata in Islandora 8
 
-> TL;DR: In Islandora 8 metadata values are stored in _fields_ attached to _entities_ (objects) which are then serialized as JSON-LD before being submitted to Fedora and/or indexed in a triple-store.
+> TL;DR: In Islandora 8, metadata is stored in _fields_ attached to _entities_ (nodes or media). Metadata values are serialized as RDF in the form of JSON-LD before being submitted to Fedora and/or indexed in a triplestore.
 
 !!! note "Drupal 8 Terminology"
-    In Drupal 8, Fields can be attached to _entity sub-types_ (e.g. Content types, Vocabularies) or _entities_ (Users, Files). For more on Fields, see ["2.3 Content Entities and Fields"](https://www.drupal.org/docs/user_guide/en/planning-data-types.html) and ["6.3 Adding Basic Fields to a Content Type"](https://www.drupal.org/docs/user_guide/en/structure-fields.html) in the Official Guide.
+    In Drupal 8, Fields can be attached to _entity sub-types_ [(formerly called _bundles_](https://www.drupal.org/project/drupal/issues/1380720) (e.g. Content types, Media types, Vocabularies) or _entities_ (Users, Files). For more on Fields, see ["2.3 Content Entities and Fields"](https://www.drupal.org/docs/user_guide/en/planning-data-types.html) and ["6.3 Adding Basic Fields to a Content Type"](https://www.drupal.org/docs/user_guide/en/structure-fields.html) in the Official Drupal Guide.
```

Comment:
Not your words (probably mine), but Files are not fieldable entities.

Reply:
It might have been my words as well; I know I touched this document quite a bit the last go-around. I think the confusion may have come from the fact that File does extend ContentEntityBase and, thus, has _fields_ but is **not** _fieldable_ (as you noted).

Reply:
haha thanks for catching that too. I had pinged Seth for confirmation as soon as I saw that. :P